Makefile.core.mk

```diff
@@ -26,15 +24,26 @@ GO ?= go
 GOARCH_LOCAL := $(TARGET_ARCH)
 GOOS_LOCAL := $(TARGET_OS)
 
+# ISTIO_IMAGE_VERSION stores the prefix used by default for the Docker images for Istio.
+# For example, a value of 1.6-dev will assume a default TAG value of 1.6-dev.<SHA>
+ISTIO_IMAGE_VERSION ?= 1.6-dev
```

Comment:
I think its 1.6-alpha

Reply:
for example: gcr.io/istio-testing/pilot:1.6-alpha.a11761a194325cc1af23e9ab53110a0369e69fde